draft-ietf-quic-transport.md

```diff
@@ -4017,6 +4017,55 @@ transport to cancel a stream in response to receipt of a STOP_SENDING frame.
 
 # Security Considerations
 
+## Handshake Denial of Service
+
+As an encrypted and authenticated transport QUIC provides a range of protections
+against denial of service.  Once the cryptographic handshake is complete, QUIC
+endpoints discard packets that are not authenticated, greatly limiting the
+ability of an attacker to interfere with existing connections.
+
+Once a connection is established QUIC endpoints might accept some
+unauthenticated ICMP packets (see {{v4-pmtud}}), but the use of these packets is
+extremely limited.  The only other type of packet that an endpoint might accept
+is a stateless reset ({{stateless-reset}}) which relies on the token being kept
+secret until it is used.
+
+During the creation of a connection, QUIC only provides protection against
+attack from off the network path.  All QUIC packets contain proof that the
+recipient saw a preceding packet from its peer.
+
+The first mechanism used is the source and destination connection IDs, which are
+required to match those set by a peer.  Except for an Initial packet and
+stateless reset, an endpoint only accepts packets that include a destination
+connection that matches a connection ID the endpoint previously chose.  This is
+the only protection offered for Version Negotiation packets.
+
+The destination connection ID in an Initial packet is selected by a client to be
+unpredictable, which serves an additional purpose.  The packets that carry the
+cryptographic handshake are protected with a key that is derived from the
+connection ID and salt specific to the QUIC version.  This allows endpoints to
+use the same process for authenticating packets that they receive as they use
+after the cryptographic handshake completes.  Packets that cannot be
+authenticated are discarded.  Protecting packets in this fashion provides a
+strong assurance that the sender of the packet saw previous packets and
+understood them.
+
+These protections are not intended to be effective against an attacker that is
+able to receive QUIC packets.  Such an attacker can potentially send packets
+that will be accepted by QUIC clients or servers.  This version of QUIC only
+attempts to detect this sort of attack.  Interference with the handshake is
```

Comment:
s/only attempts to detect/attempts to only detect/

Reply:
I've reworded this to be a little clearer (I hope).  I think that it is important to say that we don't expect endpoints to do anything other than fail when an attacker tampers with the handshake.  I realize that the lead-in to this paragraph didn't scope that properly though.  Have another look.